app/admin/device.rb

```diff
@@ -0,0 +1,32 @@
+ActiveAdmin.register Device do
+  permit_params :name, :user_id
+
+  index do
+    selectable_column
+    id_column
+    column :user_id
+    column :checkin_count do |device|
+      device.checkins.count
+    end
+    column :developer_count do |device|
+      device.developers.count
+    end
+    column :user_count do |device|
+      device.permitted_users.count
+    end
+    column :configurer do |device|
+      configurer = device.configurer
+
+      configurer.present? ? configurer.email : "Not set"
```

Comment:
Once we have a Developer table would be good to link to the developer (configurer) from this column if possible.